Cargo.toml

```diff
@@ -72,13 +71,24 @@ test-strategy = "0.3.0"
 error-stack = { version = "0.3.1", features = ["hooks"] }
 thiserror = "1.0.40"
 
-[profile.dist]
-inherits = "release"
-debug = true
-split-debuginfo = "packed"
-
 [profile.dev.package.insta]
 opt-level = 3
 
 [profile.dev.package.similar]
 opt-level = 3
+
+# The profile that 'cargo dist' will build with
+[profile.dist]
+inherits = "release"
+lto = "thin"
+
+# Config for 'cargo dist'
+[workspace.metadata.dist]
+# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.0.5"
+# The preferred Rust toolchain to use in CI (rustup toolchain syntax)
+rust-toolchain-version = "1.68"
```

Comment:
I can't think of an obvious way to do this, but it would be nice to reduce the replication of "1.68" in this file and in release.yml.

Reply:
Oh wait, it looks like this is used to generate release.yml, so maybe it's fine. Ignore me :)